.travis.yml

```
@@ -14,10 +16,19 @@ before_install:
     - make
     - sudo make install
     - nasm --version
+    - cd ..
+    - wget -O kcov.tar.gz https://github.com/SimonKagstrom/kcov/archive/v36.tar.gz
+    - tar -xvf kcov.tar.gz
+    - cd kcov-36
+    - mkdir .build && cd .build
+    - cmake .. && make && sudo make install
+    - kcov --version
+    - cargo install cargo-kcov
 script:
     - |
         cargo build --verbose &&
         cargo test --verbose &&
         cargo test --verbose --release --features=decode_test -- --ignored &&
         cargo bench --verbose &&
         cargo doc --verbose
```

Without `&&` chaining here, we might discard errors.
```suggestion
         cargo doc --verbose &&
```

---

Sounds a good idea :) Thank you!

---

(solved manually because of the habit ^^;